src/main/java/com/github/skjolberg/packing/Dimension.java

```diff
@@ -21,14 +21,14 @@ public static String encode(int width, int depth, int height) {
 	public static Dimension newInstance(int width, int depth, int height) {
 		return new Dimension(width, depth, height);
 	}
-	
+
 	protected int width; // x
 	protected int depth; // y
 	protected int height; // z
-	protected long volume;
+	long volume;
```

Comment:
Protected?

Reply:
it seems that package-protected (no keyword) was enough, it's not used in subclasses